gwcli/storage.py

```diff
@@ -378,6 +378,29 @@ def ui_command_resize(self, image_id=None, size=None):
             self.logger.error("resize needs the disk image name and new size")
             return
 
+    def ui_command_reconfigure(self, image_id=None, max_data_area_mb=None):
```

Comment:
I thought I added this comment but I guess not: I think it should use key / value pairs to override the settings so that is can be easily expanded. W/ the current approach, the second add a new field, you would be forced to specify all overrides whenever you wanted to set one (or else it would get cleared).

e.g.: ```reconfigure [image-id] [key] [value]```

Reply:
Overrides which are not specified do not get cleared. That's handled here: https://github.com/ceph/ceph-iscsi-cli/commit/0d7a3365e0a845852d0b43c05385e690989d44a8#diff-b97b10799eed7439f92d29f3e043232eR965

Also, I used `<key>=<value>` to be consistent with how other commands specify parameters.

Reply:
How does ```ui_command_reconfigure``` know which param you specified in order to clear vs which one you didn't? They both would evaluate to ```None```

Reply:
I commented on this here https://github.com/ceph/ceph-iscsi-cli/pull/99#discussion_r188575236

basically, not specifying a value resets it to its default.

Reply:
... yeah, that's ugly and that's what I'm getting at. There is plenty of president for key/value style comments in the gwcli.